pkg/cidata/cidata.TEMPLATE.d/boot/04-persistent-data-volume.sh

```diff
@@ -16,11 +16,14 @@ if [ "$(awk '$2 == "/" {print $3}' /proc/mounts)" == "tmpfs" ]; then
 	if [ -e /dev/disk/by-label/data-volume ]; then
 		# Find which disk is data volume on
 		DATA_DISK=$(blkid | grep "data-volume" | awk '{split($0,s,":"); sub(/\d$/, "", s[1]); print s[1]};')
-		# Automatically expand the data volume filesystem
-		growpart "$DATA_DISK" 1 || true
-		# Only resize when filesystem is in a healthy state
-		if e2fsck -f -p /dev/disk/by-label/data-volume; then
-			resize2fs /dev/disk/by-label/data-volume
+		# growpart command may be missing in older VMs
+		if command -v growpart >/dev/null 2>&1; then
+			# Automatically expand the data volume filesystem
+			growpart "$DATA_DISK" 1 || true
```

Comment:
Not new in this PR, but the final exit code of the script should be set to non-zero when growpart fails

Reply:
Why does it make a difference; won't the failure of `growpart` (or `resize2fs`) show up in the cloud-init log anyways?

Please file an issue if you think this is really necessary; I won't have time for it until after the holidays, and will probably forget by then...